packages/orm/src/EntityManager.ts

```diff
@@ -188,6 +188,7 @@ export type LoaderCache = Record<string, DataLoader<any, any>>;
 export class EntityManager {
   constructor(public knex: Knex) {}
 
+  private _entityIndex: Map<Function, Map<string, Entity>> = new Map();
```

Comment:
does this need to be nested?  could we just use a single level map with tagged ids?

Reply:
Oh yeah, good call. Tagged ids ftw.